beacon-chain/rpc/validator_server.go

```diff
@@ -140,6 +144,9 @@ func (vs *ValidatorServer) assignment(
 	epochStart uint64,
 ) (*pb.CommitteeAssignmentResponse_CommitteeAssignment, error) {
 
+	_, span := trace.StartSpan(ctx, "beacon-chain.rpcservice.CommitteeAssignment.assignment")
```

Comment:
why not replace ctx? without this, the span isn't passed to children.

```suggestion
	ctx, span := trace.StartSpan(ctx, "beacon-chain.rpcservice.CommitteeAssignment.assignment")
```